src/server.js

```diff
@@ -53,9 +54,14 @@ module.exports = function() {
 			log.error("Path to SSL certificate is invalid. Stopping server...");
 			process.exit();
 		}
+		if (caPath.length && !fs.existsSync(caPath)) {
+			log.error("Path to SSL ca bundle is invalid. Stopping server...");
```

Comment:
I think the point of this is just that if you specify a ca, but it doesn't exist...that's an error.

If you don't specify a ca, then it'll go through fine. But we don't want to pass through nonsense.